data/battle_ai_scripts.s

```diff
@@ -1,4 +1,4 @@
 	.section script_data, "aw", %progbits
 
 gUnknown_81D9BF4:: @ 81D9BF4
-	.incbin "baserom.gba", 0x1D9BF4, 0x20FC
+	.incbin "data/battle_ai_scripts.bin"
```

Comment:
dissassembling as bin is a bad idea, every time. Please do this right or dont do it at all.